jerry-ext/include/jerryscript-ext/arg.h

```diff
@@ -105,6 +121,8 @@ static inline jerryx_arg_t
 jerryx_arg_ignore (void);
 static inline jerryx_arg_t
 jerryx_arg_custom (void *dest, uintptr_t extra_info, jerryx_arg_transform_func_t func);
+static inline jerryx_arg_t
+jerryx_arg_object_properties (const jerryx_arg_object_props_t *object_props_p);
```

Comment:
could object args not be optional like most of the args, i.e., what about adding `jerryx_arg_optional_t opt_flag` to the arg list of this function?